scripts/stack-compose-runtime.sh

```
@@ -1433,40 +1431,51 @@ YAML
+    printf '    image: "ghcr.io/linuxserver/baseimage-alpine:3.20"\n'
+    printf '    container_name: "vpn-port-guard"\n'
     printf '    profiles:\n'
     printf '      - "ipdirect"\n'
     printf '    network_mode: "service:gluetun"\n'
     printf '    depends_on:\n'
     printf '      gluetun:\n'
     printf '        condition: "service_healthy"\n'
-    printf '      qbittorrent:\n'
-    printf '        condition: "service_started"\n'
     printf '    environment:\n'
   } >>"$dest"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)